### PR TITLE
Added BAAI/bge-small-zh-v1.5 models

### PR DESCRIPTION
**Pull request by jiangsier-xyz**

### CODE REVIEW

File: langchain4j-embeddings-bge-small-zh-v15/src/test/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15EmbeddingModelTest.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    @Test
+    void embedding_should_have_the_same_values_as_embedding_produced_by_sentence_transformers_python_lib() {
+
+        EmbeddingModel model = new BgeSmallZhV15EmbeddingModel();
+
+        Embedding embedding = model.embed("书").content();
+
+        assertThat(embedding.vector()[0]).isCloseTo(-0.0003102528f, withPercentage(1));
```
Comment (langchain4j) [inline langchain4j-embeddings-bge-small-zh-v15/src/test/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15EmbeddingModelTest.java:38]: How did you get these numbers?

**Comment by langchain4j** [comment]
Hi @jiangsier-xyz thank you!

File: langchain4j-embeddings-bge-small-zh-v15/src/test/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15EmbeddingModelTest.java (hunk 1 above)
Comment (jiangsier-xyz) [reply langchain4j-embeddings-bge-small-zh-v15/src/test/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15EmbeddingModelTest.java]: Just wrote the code according to the [documentation](https://github.com/FlagOpen/FlagEmbedding/tree/master/FlagEmbedding/llm_embedder) and tested it.

Here is my test code:
```python
import torch
from transformers import AutoTokenizer, AutoModel


def test_bge_small_zh_v15():
    queries = ['书']
    model_name = 'bge-small-zh-v15'

    # from repo on huggingface.co
    # tokenizer = AutoTokenizer.from_pretrained(f'BAAI/{model_name}')
    # model = AutoModel.from_pretrained(f'BAAI/{model_name}')

    # from local filesystem
    tokenizer = AutoTokenizer.from_pretrained(f'./{model_name}/')
    model = AutoModel.from_pretrained(f'./{model_name}/')

    query_inputs = tokenizer(queries, padding=False, return_tensors='pt')
    query_outputs = model(**query_inputs)
    query_embeddings = query_outputs.last_hidden_state[:, 0]

    with torch.no_grad():
        # Normalize
        query_embeddings = torch.nn.functional.normalize(query_embeddings, p=2, dim=1)

    print(
        f'[{queries[0]}]: {query_embeddings[0][0]} {query_embeddings[0][1]} {query_embeddings[0][510]} {query_embeddings[0][511]}')
``` 

Then, the output is
```shell
[书]: -0.00031016150023788214 0.06835819780826569 0.07749755680561066 -0.04785729572176933
```

**Comment by jiangsier-xyz** [comment]
@langchain4j Will you include it in this release? I might use this model in my application.

File: langchain4j-embeddings-bge-small-zh-v15/src/main/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15EmbeddingModel.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package dev.langchain4j.model.embedding.bge.small.zh.v15;
+
+import dev.langchain4j.model.embedding.AbstractInProcessEmbeddingModel;
+import dev.langchain4j.model.embedding.OnnxBertBiEncoder;
+import dev.langchain4j.model.embedding.PoolingMode;
+
+/**
+ * BAAI bge-small-en-v1.5 embedding model that runs within your Java application's process.
```
Comment (langchain4j) [inline langchain4j-embeddings-bge-small-zh-v15/src/main/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15EmbeddingModel.java:8]: wrong name

File: langchain4j-embeddings-bge-small-zh-v15/src/main/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15EmbeddingModel.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ * <p>
+ * Maximum length of text (in tokens) that can be embedded at once: unlimited.
+ * However, while you can embed very long texts, the quality of the embedding degrades as the text lengthens.
+ * It is recommended to embed segments of no more than 512 tokens long.
+ * <p>
+ * Embedding dimensions: 384
```
Comment (langchain4j) [inline langchain4j-embeddings-bge-small-zh-v15/src/main/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15EmbeddingModel.java:14]: wrong dimensions

File: langchain4j-embeddings-bge-small-zh-v15-q/src/main/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15QuantizedEmbeddingModel.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
+package dev.langchain4j.model.embedding.bge.small.zh.v15;
+
+import dev.langchain4j.model.embedding.AbstractInProcessEmbeddingModel;
+import dev.langchain4j.model.embedding.OnnxBertBiEncoder;
+import dev.langchain4j.model.embedding.PoolingMode;
+
+/**
+ * Quantized BAAI bge-small-en-v1.5 embedding model that runs within your Java application's process.
```
Comment (langchain4j) [inline langchain4j-embeddings-bge-small-zh-v15-q/src/main/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15QuantizedEmbeddingModel.java:8]: wrong name

File: langchain4j-embeddings-bge-small-zh-v15-q/src/main/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15QuantizedEmbeddingModel.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ * <p>
+ * Maximum length of text (in tokens) that can be embedded at once: unlimited.
+ * However, while you can embed very long texts, the quality of the embedding degrades as the text lengthens.
+ * It is recommended to embed segments of no more than 512 tokens long.
+ * <p>
+ * Embedding dimensions: 384
```
Comment (langchain4j) [inline langchain4j-embeddings-bge-small-zh-v15-q/src/main/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15QuantizedEmbeddingModel.java:14]: wrong dimensions

File: langchain4j-embeddings-bge-small-zh-v15-q/src/main/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15QuantizedEmbeddingModel.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+ * <p>
+ * It is recommended to add "Represent this sentence for searching relevant passages:" prefix to a query.
+ * <p>
+ * More details <a href="https://huggingface.co/BAAI/bge-small-en-v1.5">here</a>
```
Comment (langchain4j) [inline langchain4j-embeddings-bge-small-zh-v15-q/src/main/java/dev/langchain4j/model/embedding/bge/small/zh/v15/BgeSmallZhV15QuantizedEmbeddingModel.java:18]: wrong link

**Review by langchain4j — COMMENTED** [review]
Thank you! Please fix copy-paste errors

**Comment by jiangsier-xyz** [comment]
> Thank you! Please fix copy-paste errors

My oversight. All done.

**Pull request merged**